resources/assets/less/bootstrap-variables.less

```diff
@@ -53,7 +53,7 @@
 @input-color: @gray-dark;
 @input-border-focus: @pink;
 
-@screen-sm-min: 900px; // sync with osu.isDesktop
+@screen-sm-min: 900px; // sync with osu.isDesktop, react/beatmaps/main.coffee
```

Comment:
should probably figure out a way to import this into scripts

Reply:
why not use the `isDesktop` function? ಠ_ಠ

Reply:
owait :D